### ISSUE TITLE
RIDE GUI App failing to start due to setlocale() error

### ISSUE THREAD

**Issue by FransPeterson** [robotframework/RIDE#2060]
Hi all,

I'm having an issue with RIDE when trying to run the GUI APP, it's failing to start with below tracing: 

>ride.py

```
Traceback (most recent call last):
  File "appdata\local\programs\python\python38-32\lib\site-packages\robotide\application\application.py", line 55, in OnInit
    self.frame = RideFrame(self, self._controller)
  File "appdata\local\programs\python\python38-32\lib\site-packages\robotide\ui\mainframe.py", line 166, in __init__
    self._init_ui()
  File "\appdata\local\programs\python\python38-32\lib\site-packages\robotide\ui\mainframe.py", line 267, in _init_ui
    self.tree = Tree(self, self.actions,
  File "appdata\local\programs\python\python38-32\lib\site-packages\robotide\ui\tree.py", line 85, in __init__
    self._images = TreeImageList()
  File "appdata\local\programs\python\python38-32\lib\site-packages\robotide\ui\images.py", line 42, in __init__
    TestCaseController: _TreeImage(self, 'robot.png'),
  File "appdata\local\programs\python\python38-32\lib\site-packages\robotide\ui\images.py", line 80, in __init__
    self.normal = self._get_image(image_list, normal)
  File "appdata\local\programs\python\python38-32\lib\site-packages\robotide\ui\images.py", line 91, in _get_image
    img = wx.Image(path, wx.BITMAP_TYPE_PNG).ConvertToBitmap()
wx._core.wxAssertionError: C++ assertion "strcmp(setlocale(LC_ALL, NULL), "C") == 0" failed at ..\..\src\common\intl.cpp(1579) in wxLocale::GetInfo(): You probably called setlocale() directly instead of using wxLocale and now there is a mismatch between C/C++ and Windows locale.
Things are going to break, please only change locale by creating wxLocale objects to avoid this!
OnInit returned false, exiting...
Error in atexit._run_exitfuncs:
wx._core.wxAssertionError: C++ assertion "GetEventHandler() == this" failed at ..\..\src\common\wincmn.cpp(478) in wxWindowBase::~wxWindowBase(): any pushed event handlers must have been removed

```
Versions used:
Robot Framework 3.1.2 (Python 3.8.0 on win32)
RIDE 1.7.3.1
WXPython 4.0.7.post2

**Comment by HelioGuilherme66** [comment]
We have put a warning on the [README](https://github.com/robotframework/RIDE/blob/master/README.rest) on frontpage:

**Attention, don't install RIDE on Python 3.8 (wait for version 2.0)**

**Comment by FransPeterson** [comment]
OK, will revert back to Python 2.

**Comment by HelioGuilherme66** [comment]
You can use python 3.7.

**Comment by HelioGuilherme66** [comment]
Fixed by #2128

**Comment by apoom15** [comment]
Hi,
i have install ride on top of Python 2.7.18. 
ride version1.7.4.1. But still i am  getting this error
Could you please help?
